src/main/java/com/powsybl/openloadflow/network/impl/LfNetworkLoaderImpl.java

```diff
@@ -1049,6 +1170,19 @@ private void createAutomationSystems(Network network, LfNetwork lfNetwork) {
         }
     }
 
+    static void checkDuplicatedAreas(List<LfNetwork> lfNetworks) {
+        List<String> duplicatedAreas = lfNetworks.stream()
+                .flatMap(lfNetwork -> lfNetwork.getAreas().stream())
+                .collect(Collectors.groupingBy(LfArea::getId, Collectors.counting()))
+                .entrySet().stream()
+                .filter(e -> e.getValue() > 1)
+                .map(Map.Entry::getKey)
+                .toList();
+        if (!duplicatedAreas.isEmpty()) {
+            throw new PowsyblException("Areas with ids " + duplicatedAreas + " are present in more than one LfNetwork. Load flow computation with area interchange control is not supported in this case.");
+        }
+    }
```

Comment:
with this, in practice, you are going to throw for all real case networks which always have small portions here and there.

The correct check is verifying that each area has all its boundaries are in the same LfNetwork.

Reply:
I updated it in 164e8993a75d730a8c72dbfe3c35d2227e1bb7e6 to check the boundaries.
For the moment it still throws.

Are you sure that we want just a warn ? 
We convert only the AIC areas, which means that with boundaries of a same area beeing in diffrent islands, it s pretty guaranteed that we will have weird results.

Reply:
I need to think about it ... could be either automatically disbabling AIC, or returning a status failed for the involved areas.

Reply:
Best way to do I think is to create a new enum constant in `com.powsybl.openloadflow.network.LfNetwork.Validity` e.g. 

`INVALID_FRAGMENTED_AREA("Network contains area(s) spanning multiple components")`

Reply:
Actually the best would be to not perform interchange control, but only for the fragmented areas, i.e. area that are not fragmented still perform AIC